.circleci/scripts/setup_ci_environment.sh

```diff
@@ -47,31 +47,42 @@ else
   sudo rm -rf /etc/apt/sources.list.d/nvidia-docker.list
 fi
 
+add_to_env_file() {
```

Comment:
Moving this to a function makes it easier to refactor if we decide to change where we want to store our `env-file`